permissions/helpers.py

```diff
@@ -166,7 +169,11 @@ def can_manage_service_sectors_applications(
 def can_modify_application(user: User, application: Application) -> bool:
     return (
         is_superuser(user)
-        or application.user == user
+        or (
+            application.user == user
+            and datetime.now(tz=UTC)
```

Comment:
Would using 

`timezone.get_default_timezone()`

be safer since it'd always get the timezone from settings (although it's UTC atm, but i guess could theoretically change)

Reply:
good point - I'll do the change :+1: